hbase-thrift/src/main/resources/org/apache/hadoop/hbase/thrift2/hbase.thrift

```diff
@@ -940,6 +940,9 @@ service THBaseService {
    * region gets splitted, the api may return false.
    *
    * @return true if table is available, false if not
+   *
+   * @deprecated Since 2.0.0. Will be removed in 3.0.0.
```

Comment:
It should be 2.2.0? I mean you need to say that the same method in Table interface has been deprecated since 2.0.0, so we will remove it in 3.0.0 release.

Reply:
ok ,I have amended